tools/experiments/experiments-config.js

```diff
@@ -220,6 +220,11 @@ export const EXPERIMENTS = [
     name: 'Client-side granular consent',
     spec: 'https://github.com/ampproject/amphtml/issues/31607',
   },
+  {
+    id: 'amp-story-inline-page-attachment-v2',
```

Comment:
We should make the experiment global to all changes to page attachments, so we can launch everything at once when ready.